index.html

```diff
@@ -1,9 +1,184 @@
+            egestas. Vestibulum
+            tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
+            amet quam egestas
+            semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo. Quisque sit amet
+            est et sapien
+            ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, commodo vitae, ornare sit
+            amet, wisi. Aenean
+            fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus
+            lacus enim ac dui. Donec
+            non enim in turpis pulvinar facilisis. Ut felis. Praesent dapibus, neque id cursus
+            faucibus, tortor neque
+            egestas augue, eu vulputate magna eros eu erat. Aliquam erat volutpat. Nam dui mi,
+            tincidunt quis, accumsan
+            porttitor, facilisis luctus, metus</p>
+        <q>Vestibulum tortor quam</q>
+        <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis
+            egestas. Vestibulum
+            tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
+            amet quam egestas
+            semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo. Quisque sit amet
+            est et sapien
+            ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, commodo vitae, ornare sit
+            amet, wisi. Aenean
+            fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus
+            lacus enim ac dui. Donec
+            non enim in turpis pulvinar facilisis. Ut felis. Praesent dapibus, neque id cursus
+            faucibus, tortor neque
+            egestas augue, eu vulputate magna eros eu erat. Aliquam erat volutpat. Nam dui mi,
+            tincidunt quis, accumsan
+            porttitor, facilisis luctus, metus</p>
+    </article>
+    <article>
+        <h2>Article 3</h2>
+        <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis
+            egestas. Vestibulum
+            tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
+            amet quam egestas
+            semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo. Quisque sit amet
+            est et sapien
+            ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, commodo vitae, ornare sit
+            amet, wisi. Aenean
+            fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus
+            lacus enim ac dui. Donec
+            non enim in turpis pulvinar facilisis. Ut felis. Praesent dapibus, neque id cursus
+            faucibus, tortor neque
+            egestas augue, eu vulputate magna eros eu erat. Aliquam erat volutpat. Nam dui mi,
+            tincidunt quis, accumsan
+            porttitor, facilisis luctus, metus</p>
+        <img
```

Comment:
я знаю про alt, но зачем title?

Reply:
http://vsemproblemam.net/seo/izobrahzenija-i-ssylki-atributy-alt-i-title.html
1 статья в гугле между прочим)

Reply:
Очень странный источник :) и то ссылается на "SEO-экспертов".
В 2 англоязычных статьях нашел примерно такое:
> The image title attribute doesn’t have any impact on keyword rankings because search engines don’t crawl them, so if you had to choose between optimizing alt text and titles, you should choose the former every time. That said, image titles are displayed when your mouse hovers over an image in some browsers (like Firefox and Opera) so it does improve user experience to have a proper title set.

Т.е. и на SEO оно влияния не оказывает. Только тултип.

Reply:
> Очень странный источник :) и то ссылается на "SEO-экспертов".
> В 2 англоязычных статьях нашел примерно такое:
> 
> > The image title attribute doesn’t have any impact on keyword rankings because search engines don’t crawl them, so if you had to choose between optimizing alt text and titles, you should choose the former every time. That said, image titles are displayed when your mouse hovers over an image in some browsers (like Firefox and Opera) so it does improve user experience to have a proper title set.
> 
> Т.е. и на SEO оно влияния не оказывает. Только тултип.

Даже в той цитате что ты привел сказано главное "it does improve user experience" =)

Reply:
Суть то в том, что атрибут не обязательный, в отличие от alt.
Нужны или нет тултипы - другой вопрос, в задании нет :)